src/main/resources/assets/eln/lang/en_US.lang

```diff
@@ -431,7 +431,7 @@ Player_crankable_shaft=Player crankable shaft
 Can_rotate_slowly=Can rotate slowly
 
 # ./src/main/java/mods/eln/mechanical/Generator.kt
-Converts_mechanical_energy_into_electricity,_or_(badly)_vice_versa.=将机械能转换为电能，或反之(低效)。
+Converts_mechanical_energy_into_electricity,_or_(badly)_vice_versa.=Converts_mechanical_energy_into_electricity,_or_(badly)_vice_versa.
```

Comment:
`Converts mechanical energy into electricity, or (badly) vice versa.` is what should be on the right side of the equal sign. The left side of the translation file is the "tag" that the translator uses (and it escapes spaces and certain variables) and the right side is the final translation plus the variable tags.